app/src/org/commcare/activities/FormEntryActivity.java

```diff
@@ -212,22 +209,6 @@ public void onCreateSessionSafe(Bundle savedInstanceState) {
         }
     }
 
-    @Override
```

Comment:
Ohh strange. Is this not needed anymore?

Reply:
Two reasons why I removed this - 
1. `onMenuItemSelected` is final in `AppCompatActivity` demanding a change here
2. This will only be applicable for api v17 devices and quick look through analytics suggest that we don't have any users on api-17 devices.

Reply:
> This will only be applicable for api v17 devices and quick look through analytics suggest that we don't have any users on api-17 devices.

This doesn't seem true anymore. Suddently there is an influx of devices on v17. Unfortunately BS doesn't have any devices less than 4.4 for me to test any new implementations of this method. Therefore [committed an alternative change](https://github.com/dimagi/commcare-android/pull/2342/commits/2d05d5199162c45d7c8f307f5f2e8cb288579ada) to this which removes setting spannable to the Context menu. This was redundant there as we don't have any formatting anyway in that context menu and just plain strings. So there is no use of using Spannnables there.  Also love how useful the links to tickets in PRs are. I was able to track this down to http://manage.dimagi.com/default.asp?174626